qutip/qobj.py

```diff
@@ -1395,6 +1395,8 @@ def tidyup(self, atol=settings.auto_tidyup_atol):
             Quantum object with small elements removed.
 
         """
+        atol = atol or settings.auto_tidyup_atol
```

Comment:
```suggestion
        if atol is None:
            atol = settings.auto_tidyup_atol
```
The `atol or settings.auto_tidyup_atol` will prevent using `atol=0.0`. Perhaps not the most important use case, but it worked before and I can see people relying on it rather than writing `if atol: obj = obj.tidyup(atol=atol` in their own code.